website/docs/api/modules/dids.md

```diff
@@ -12,6 +14,149 @@ A simple library to interact with DIDs that conform to the DID-provider interfac
 npm install dids
 ```
 
+## API
+
+[API documentation](https://ceramicnetwork.github.io/js-did/classes/did.html)
```

Comment:
Doesn't seem to be a correct link, for the DID class it should be https://did.js.org/docs/api/classes/dids.DID